solidity/test/isms/DomainRoutingIsm.t.sol

```diff
@@ -72,15 +75,18 @@ contract DomainRoutingIsmTest is Test {
         ism.set(domain, deployTestIsm(seed));
 
         bytes memory metadata = getMetadata(domain);
+        uint256 gasBefore = gasleft();
         assertTrue(ism.verify(metadata, MessageUtils.build(domain)));
+        uint256 gasAfter = gasleft();
+        console.log("Overhead gas usage: %d", gasBefore - gasAfter);
     }
 
     function testVerifyNoIsm(uint32 domain, bytes32 seed) public virtual {
         vm.assume(domain > 0);
         ism.set(domain, deployTestIsm(seed));
 
         bytes memory metadata = getMetadata(domain);
-        vm.expectRevert("No ISM found for origin domain");
+        vm.expectRevert();
```

Comment:
Do you want to make a forge issue for expecting custom strings?

Reply:
https://github.com/foundry-rs/foundry/issues/5941